Tests/Controller/LocaleControllerTest.php

```diff
@@ -31,7 +35,6 @@ public function testControllerThrowsException()
         $request->setLocale('en');
         $localeController = new LocaleController($this->getRouterMock(), $metaValidatorMock);
 
-        $this->setExpectedException('\InvalidArgumentException');
```

Comment:
can we set a minimum phpunit version that has `setExpectedException` ? its much better to expect the exception at the place we want it to happen, rather than the annotation on the test.